{"name":"givenInvalidData_whenApplyingRule_itIsEvaluatedAsInvalid()","id":"[engine:junit-jupiter]/[class:be.sddevelopment.validation.ConstrainableTest]/[method:givenInvalidData_whenApplyingRule_itIsEvaluatedAsInvalid()]","scenarioId":"[engine:junit-jupiter]/[class:be.sddevelopment.validation.ConstrainableTest]","testCaseName":"be.sddevelopment.validation.ConstrainableTest","methodName":"givenInvalidData_whenApplyingRule_itIsEvaluatedAsInvalid","userStory":{"id":"be.sddevelopment.validation.ConstrainableTest","storyName":"ConstrainableTest","displayName":"ConstrainableTest","storyClassName":"be.sddevelopment.validation.ConstrainableTest","path":"ConstrainableTest","pathElements":[{"name":"ConstrainableTest","description":"ConstrainableTest"}],"narrative":"","type":"feature"},"title":"Given invalid data when applying rule it is evaluated as invalid","tags":[{"name":"ConstrainableTest","type":"feature","displayName":"ConstrainableTest"}],"startTime":"2023-12-28T06:25:24.221708015+01:00[Europe/Brussels]","duration":3,"projectKey":"","driver":"","isManualTestingUpToDate":false,"manual":false,"testSource":"JUnit5","order":0,"result":"SUCCESS"}